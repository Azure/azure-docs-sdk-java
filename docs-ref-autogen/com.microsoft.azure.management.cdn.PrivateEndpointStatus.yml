### YamlMime:JavaType
uid: "com.microsoft.azure.management.cdn.PrivateEndpointStatus"
fullName: "com.microsoft.azure.management.cdn.PrivateEndpointStatus"
name: "PrivateEndpointStatus"
nameWithType: "PrivateEndpointStatus"
summary: "Defines values for PrivateEndpointStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.microsoft.rest.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)"
- "com.microsoft.rest.ExpandableStringEnum.hashCode()"
- "com.microsoft.rest.ExpandableStringEnum.toString()"
- "com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointStatus extends ExpandableStringEnum<PrivateEndpointStatus>"
constructors:
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.PrivateEndpointStatus()"
fields:
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.APPROVED"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.DISCONNECTED"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.PENDING"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.REJECTED"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.TIMEOUT"
methods:
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.fromString(java.lang.String)"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus.values()"
type: "class"
metadata: {}
package: "com.microsoft.azure.management.cdn"
artifact: com.microsoft.azure:azure-mgmt-cdn:1.41.1
