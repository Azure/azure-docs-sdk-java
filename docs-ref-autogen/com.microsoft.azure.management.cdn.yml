### YamlMime:JavaPackage
uid: "com.microsoft.azure.management.cdn"
fullName: "com.microsoft.azure.management.cdn"
name: "com.microsoft.azure.management.cdn"
summary: "This package contains the classes for CdnManagementClient.\n Cdn Management Client."
classes:
- "com.microsoft.azure.management.cdn.AFDDomainHttpsParameters"
- "com.microsoft.azure.management.cdn.AFDDomainUpdateParameters"
- "com.microsoft.azure.management.cdn.AFDEndpointProtocols"
- "com.microsoft.azure.management.cdn.AFDEndpointUpdateParameters"
- "com.microsoft.azure.management.cdn.AFDOriginGroupUpdateParameters"
- "com.microsoft.azure.management.cdn.AFDOriginUpdateParameters"
- "com.microsoft.azure.management.cdn.AFDStateProperties"
- "com.microsoft.azure.management.cdn.ActionType"
- "com.microsoft.azure.management.cdn.AfdCertificateType"
- "com.microsoft.azure.management.cdn.AfdErrorResponse"
- "com.microsoft.azure.management.cdn.AfdErrorResponseException"
- "com.microsoft.azure.management.cdn.AfdProvisioningState"
- "com.microsoft.azure.management.cdn.AfdPurgeParameters"
- "com.microsoft.azure.management.cdn.Algorithm"
- "com.microsoft.azure.management.cdn.CacheBehavior"
- "com.microsoft.azure.management.cdn.CacheExpirationActionParameters"
- "com.microsoft.azure.management.cdn.CacheKeyQueryStringActionParameters"
- "com.microsoft.azure.management.cdn.CdnCertificateSourceParameters"
- "com.microsoft.azure.management.cdn.CdnManagedHttpsParameters"
- "com.microsoft.azure.management.cdn.CdnWebApplicationFirewallPolicyPatchParameters"
- "com.microsoft.azure.management.cdn.Certificate"
- "com.microsoft.azure.management.cdn.CertificateType"
- "com.microsoft.azure.management.cdn.CheckNameAvailabilityInput"
- "com.microsoft.azure.management.cdn.CheckNameAvailabilityResult"
- "com.microsoft.azure.management.cdn.CidrIpAddress"
- "com.microsoft.azure.management.cdn.CompressionSettings"
- "com.microsoft.azure.management.cdn.ContinentsResponseContinentsItem"
- "com.microsoft.azure.management.cdn.ContinentsResponseCountryOrRegionsItem"
- "com.microsoft.azure.management.cdn.CookiesMatchConditionParameters"
- "com.microsoft.azure.management.cdn.CookiesOperator"
- "com.microsoft.azure.management.cdn.CustomDomainHttpsParameters"
- "com.microsoft.azure.management.cdn.CustomDomainParameters"
- "com.microsoft.azure.management.cdn.CustomDomainResourceState"
- "com.microsoft.azure.management.cdn.CustomDomainValidationResult"
- "com.microsoft.azure.management.cdn.CustomHttpsProvisioningState"
- "com.microsoft.azure.management.cdn.CustomHttpsProvisioningSubstate"
- "com.microsoft.azure.management.cdn.CustomRule"
- "com.microsoft.azure.management.cdn.CustomRuleEnabledState"
- "com.microsoft.azure.management.cdn.CustomRuleList"
- "com.microsoft.azure.management.cdn.CustomerCertificate"
- "com.microsoft.azure.management.cdn.CustomerCertificateParameters"
- "com.microsoft.azure.management.cdn.DeepCreatedOrigin"
- "com.microsoft.azure.management.cdn.DeepCreatedOriginGroup"
- "com.microsoft.azure.management.cdn.DeliveryRule"
- "com.microsoft.azure.management.cdn.DeliveryRuleAction"
- "com.microsoft.azure.management.cdn.DeliveryRuleCacheExpirationAction"
- "com.microsoft.azure.management.cdn.DeliveryRuleCacheKeyQueryStringAction"
- "com.microsoft.azure.management.cdn.DeliveryRuleCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleCookiesCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleHttpVersionCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleIsDeviceCondition"
- "com.microsoft.azure.management.cdn.DeliveryRulePostArgsCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleQueryStringCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRemoteAddressCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestBodyCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestHeaderAction"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestHeaderCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestMethodCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestSchemeCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleRequestUriCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleResponseHeaderAction"
- "com.microsoft.azure.management.cdn.DeliveryRuleUrlFileExtensionCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleUrlFileNameCondition"
- "com.microsoft.azure.management.cdn.DeliveryRuleUrlPathCondition"
- "com.microsoft.azure.management.cdn.DeploymentStatus"
- "com.microsoft.azure.management.cdn.DestinationProtocol"
- "com.microsoft.azure.management.cdn.DomainValidationProperties"
- "com.microsoft.azure.management.cdn.DomainValidationState"
- "com.microsoft.azure.management.cdn.EdgeNode"
- "com.microsoft.azure.management.cdn.EnabledState"
- "com.microsoft.azure.management.cdn.EndpointPropertiesUpdateParametersDeliveryPolicy"
- "com.microsoft.azure.management.cdn.EndpointPropertiesUpdateParametersWebApplicationFirewallPolicyLink"
- "com.microsoft.azure.management.cdn.EndpointResource"
- "com.microsoft.azure.management.cdn.EndpointResourceState"
- "com.microsoft.azure.management.cdn.EndpointUpdateParameters"
- "com.microsoft.azure.management.cdn.ErrorResponse"
- "com.microsoft.azure.management.cdn.ErrorResponseException"
- "com.microsoft.azure.management.cdn.ForwardingProtocol"
- "com.microsoft.azure.management.cdn.GeoFilter"
- "com.microsoft.azure.management.cdn.HeaderAction"
- "com.microsoft.azure.management.cdn.HeaderActionParameters"
- "com.microsoft.azure.management.cdn.HealthProbeParameters"
- "com.microsoft.azure.management.cdn.HttpErrorRangeParameters"
- "com.microsoft.azure.management.cdn.HttpVersionMatchConditionParameters"
- "com.microsoft.azure.management.cdn.HttpsRedirect"
- "com.microsoft.azure.management.cdn.IdentityType"
- "com.microsoft.azure.management.cdn.IpAddressGroup"
- "com.microsoft.azure.management.cdn.IsDeviceMatchConditionParameters"
- "com.microsoft.azure.management.cdn.KeyVaultCertificateSourceParameters"
- "com.microsoft.azure.management.cdn.KeyVaultSigningKeyParameters"
- "com.microsoft.azure.management.cdn.LinkToDefaultDomain"
- "com.microsoft.azure.management.cdn.LoadBalancingSettingsParameters"
- "com.microsoft.azure.management.cdn.LoadParameters"
- "com.microsoft.azure.management.cdn.LogMetric"
- "com.microsoft.azure.management.cdn.LogMetricsGranularity"
- "com.microsoft.azure.management.cdn.LogMetricsGroupBy"
- "com.microsoft.azure.management.cdn.LogRanking"
- "com.microsoft.azure.management.cdn.LogRankingMetric"
- "com.microsoft.azure.management.cdn.ManagedCertificate"
- "com.microsoft.azure.management.cdn.ManagedCertificateParameters"
- "com.microsoft.azure.management.cdn.ManagedRuleDefinition"
- "com.microsoft.azure.management.cdn.ManagedRuleEnabledState"
- "com.microsoft.azure.management.cdn.ManagedRuleGroupDefinition"
- "com.microsoft.azure.management.cdn.ManagedRuleGroupOverride"
- "com.microsoft.azure.management.cdn.ManagedRuleOverride"
- "com.microsoft.azure.management.cdn.ManagedRuleSetList"
- "com.microsoft.azure.management.cdn.MatchCondition"
- "com.microsoft.azure.management.cdn.MatchProcessingBehavior"
- "com.microsoft.azure.management.cdn.MatchVariable"
- "com.microsoft.azure.management.cdn.MetricsResponseSeriesItem"
- "com.microsoft.azure.management.cdn.MetricsResponseSeriesItemDataItem"
- "com.microsoft.azure.management.cdn.MetricsResponseSeriesItemGroupsItem"
- "com.microsoft.azure.management.cdn.Operation"
- "com.microsoft.azure.management.cdn.OperationDisplay"
- "com.microsoft.azure.management.cdn.Operator"
- "com.microsoft.azure.management.cdn.OptimizationType"
- "com.microsoft.azure.management.cdn.OriginGroupOverrideAction"
- "com.microsoft.azure.management.cdn.OriginGroupOverrideActionParameters"
- "com.microsoft.azure.management.cdn.OriginGroupResourceState"
- "com.microsoft.azure.management.cdn.OriginGroupUpdateParameters"
- "com.microsoft.azure.management.cdn.OriginResourceState"
- "com.microsoft.azure.management.cdn.OriginUpdateParameters"
- "com.microsoft.azure.management.cdn.ParamIndicator"
- "com.microsoft.azure.management.cdn.PolicyEnabledState"
- "com.microsoft.azure.management.cdn.PolicyMode"
- "com.microsoft.azure.management.cdn.PolicyResourceState"
- "com.microsoft.azure.management.cdn.PolicySettings"
- "com.microsoft.azure.management.cdn.PostArgsMatchConditionParameters"
- "com.microsoft.azure.management.cdn.PostArgsOperator"
- "com.microsoft.azure.management.cdn.PrivateEndpointStatus"
- "com.microsoft.azure.management.cdn.ProfileResourceState"
- "com.microsoft.azure.management.cdn.ProfileUpdateParameters"
- "com.microsoft.azure.management.cdn.ProtocolType"
- "com.microsoft.azure.management.cdn.ProvisioningState"
- "com.microsoft.azure.management.cdn.PurgeParameters"
- "com.microsoft.azure.management.cdn.QueryStringBehavior"
- "com.microsoft.azure.management.cdn.QueryStringMatchConditionParameters"
- "com.microsoft.azure.management.cdn.QueryStringOperator"
- "com.microsoft.azure.management.cdn.RankingsResponseTablesItem"
- "com.microsoft.azure.management.cdn.RankingsResponseTablesItemDataItem"
- "com.microsoft.azure.management.cdn.RankingsResponseTablesItemDataItemMetricsItem"
- "com.microsoft.azure.management.cdn.RateLimitRule"
- "com.microsoft.azure.management.cdn.RateLimitRuleList"
- "com.microsoft.azure.management.cdn.RedirectType"
- "com.microsoft.azure.management.cdn.RemoteAddressMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RemoteAddressOperator"
- "com.microsoft.azure.management.cdn.RequestBodyMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RequestBodyOperator"
- "com.microsoft.azure.management.cdn.RequestHeaderMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RequestHeaderOperator"
- "com.microsoft.azure.management.cdn.RequestMethodMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RequestSchemeMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RequestUriMatchConditionParameters"
- "com.microsoft.azure.management.cdn.RequestUriOperator"
- "com.microsoft.azure.management.cdn.ResourceReference"
- "com.microsoft.azure.management.cdn.ResourceUsage"
- "com.microsoft.azure.management.cdn.ResourcesResponseCustomDomainsItem"
- "com.microsoft.azure.management.cdn.ResourcesResponseEndpointsItem"
- "com.microsoft.azure.management.cdn.ResourcesResponseEndpointsItemCustomDomainsItem"
- "com.microsoft.azure.management.cdn.ResponseBasedOriginErrorDetectionParameters"
- "com.microsoft.azure.management.cdn.RouteUpdateParameters"
- "com.microsoft.azure.management.cdn.RuleUpdateParameters"
- "com.microsoft.azure.management.cdn.SecretParameters"
- "com.microsoft.azure.management.cdn.SecretProperties"
- "com.microsoft.azure.management.cdn.SecurityPolicyParameters"
- "com.microsoft.azure.management.cdn.SecurityPolicyProperties"
- "com.microsoft.azure.management.cdn.SecurityPolicyWebApplicationFirewallAssociation"
- "com.microsoft.azure.management.cdn.SecurityPolicyWebApplicationFirewallParameters"
- "com.microsoft.azure.management.cdn.SharedPrivateLinkResourceProperties"
- "com.microsoft.azure.management.cdn.Sku"
- "com.microsoft.azure.management.cdn.SkuName"
- "com.microsoft.azure.management.cdn.Status"
- "com.microsoft.azure.management.cdn.SystemData"
- "com.microsoft.azure.management.cdn.Transform"
- "com.microsoft.azure.management.cdn.TransformType"
- "com.microsoft.azure.management.cdn.UrlFileExtensionMatchConditionParameters"
- "com.microsoft.azure.management.cdn.UrlFileExtensionOperator"
- "com.microsoft.azure.management.cdn.UrlFileNameMatchConditionParameters"
- "com.microsoft.azure.management.cdn.UrlFileNameOperator"
- "com.microsoft.azure.management.cdn.UrlPathMatchConditionParameters"
- "com.microsoft.azure.management.cdn.UrlPathOperator"
- "com.microsoft.azure.management.cdn.UrlRedirectAction"
- "com.microsoft.azure.management.cdn.UrlRedirectActionParameters"
- "com.microsoft.azure.management.cdn.UrlRewriteAction"
- "com.microsoft.azure.management.cdn.UrlRewriteActionParameters"
- "com.microsoft.azure.management.cdn.UrlSigningAction"
- "com.microsoft.azure.management.cdn.UrlSigningActionParameters"
- "com.microsoft.azure.management.cdn.UrlSigningKey"
- "com.microsoft.azure.management.cdn.UrlSigningKeyParameters"
- "com.microsoft.azure.management.cdn.UrlSigningParamIdentifier"
- "com.microsoft.azure.management.cdn.UsageName"
- "com.microsoft.azure.management.cdn.UserManagedHttpsParameters"
- "com.microsoft.azure.management.cdn.ValidateCustomDomainInput"
- "com.microsoft.azure.management.cdn.ValidateProbeInput"
- "com.microsoft.azure.management.cdn.ValidateSecretInput"
- "com.microsoft.azure.management.cdn.ValidateSecretType"
- "com.microsoft.azure.management.cdn.WafAction"
- "com.microsoft.azure.management.cdn.WafGranularity"
- "com.microsoft.azure.management.cdn.WafMetric"
- "com.microsoft.azure.management.cdn.WafMetricsResponseSeriesItem"
- "com.microsoft.azure.management.cdn.WafMetricsResponseSeriesItemDataItem"
- "com.microsoft.azure.management.cdn.WafMetricsResponseSeriesItemGroupsItem"
- "com.microsoft.azure.management.cdn.WafRankingGroupBy"
- "com.microsoft.azure.management.cdn.WafRankingType"
- "com.microsoft.azure.management.cdn.WafRankingsResponseDataItem"
- "com.microsoft.azure.management.cdn.WafRankingsResponseDataItemMetricsItem"
- "com.microsoft.azure.management.cdn.WafRuleType"
enums:
- "com.microsoft.azure.management.cdn.AfdMinimumTlsVersion"
- "com.microsoft.azure.management.cdn.AfdQueryStringCachingBehavior"
- "com.microsoft.azure.management.cdn.GeoFilterActions"
- "com.microsoft.azure.management.cdn.HealthProbeRequestType"
- "com.microsoft.azure.management.cdn.MinimumTlsVersion"
- "com.microsoft.azure.management.cdn.ProbeProtocol"
- "com.microsoft.azure.management.cdn.QueryStringCachingBehavior"
- "com.microsoft.azure.management.cdn.ResourceType"
- "com.microsoft.azure.management.cdn.ResponseBasedDetectedErrorTypes"
- "com.microsoft.azure.management.cdn.SharedPrivateLinkResourceStatus"
interfaces:
- "com.microsoft.azure.management.cdn.CdnEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.AttachablePremium"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.AttachableStandard"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.Blank"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.Blank.PremiumEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.Blank.StandardEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.WithPremiumAttach"
- "com.microsoft.azure.management.cdn.CdnEndpoint.DefinitionStages.WithStandardAttach"
- "com.microsoft.azure.management.cdn.CdnEndpoint.Update"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.AttachablePremium"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.AttachableStandard"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.Blank"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.Blank.PremiumEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.Blank.StandardEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.WithPremiumAttach"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateDefinitionStages.WithStandardAttach"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdatePremiumEndpoint"
- "com.microsoft.azure.management.cdn.CdnEndpoint.UpdateStandardEndpoint"
- "com.microsoft.azure.management.cdn.CdnProfile"
- "com.microsoft.azure.management.cdn.CdnProfile.Definition"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.Blank"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.WithGroup"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.WithPremiumVerizonCreate"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.WithSku"
- "com.microsoft.azure.management.cdn.CdnProfile.DefinitionStages.WithStandardCreate"
- "com.microsoft.azure.management.cdn.CdnProfile.Update"
- "com.microsoft.azure.management.cdn.CdnProfile.UpdateStages"
- "com.microsoft.azure.management.cdn.CdnProfile.UpdateStages.WithEndpoint"
- "com.microsoft.azure.management.cdn.CdnProfiles"
metadata: {}
package: "com.microsoft.azure.management.cdn"
artifact: com.microsoft.azure:azure-mgmt-cdn:1.41.1
