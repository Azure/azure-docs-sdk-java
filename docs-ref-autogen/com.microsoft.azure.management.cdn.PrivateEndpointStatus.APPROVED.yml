### YamlMime:JavaMember
uid: "com.microsoft.azure.management.cdn.PrivateEndpointStatus.APPROVED*"
fullName: "com.microsoft.azure.management.cdn.PrivateEndpointStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.cdn.PrivateEndpointStatus.APPROVED"
  fullName: "com.microsoft.azure.management.cdn.PrivateEndpointStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointStatus."
  syntax: "public static final PrivateEndpointStatus APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.cdn"
artifact: com.microsoft.azure:azure-mgmt-cdn:1.41.1
